products/views.py

```diff
@@ -1,3 +1,24 @@
-from django.shortcuts import render
+from django.views import View
+from django.http import JsonResponse
 
-# Create your views here.
+from .models import Product
+
+class ProductSearchView(View):
+    def get(self, request):
+
+        page          = int(request.GET.get('page', 1))
+        limit         = int(request.GET.get('limit', 5))
+        offset        = (page-1) * limit
```

Comment:
API는 페이지 기준으로 pagination을 하는 것 보다는 `offset`, `limit`으로 페이지네이션 하는 것이 좋습니다.
```suggestion
        page          = int(request.GET.get('page', 1))
        limit         = int(request.GET.get('limit', 5))
        offset        = (page-1) * limit
```